packages/bezier-react/src/components/Forms/Inputs/TextField/TextField.tsx

```diff
@@ -117,7 +118,7 @@ forwardedRef: Ref<TextFieldRef>,
   ), [value])
 
   const activeInput = !disabled && !readOnly
-  const activeClear = activeInput && allowClear
+  const activeClear = activeInput && allowClear && !!normalizedValue
```

Comment:
```suggestion
  const activeClear = activeInput && allowClear && !isEmpty(normalizedValue)
```

빈 문자열까지 고려하면 empty 체크를 하는 게 더 명시적일 거 같습니다.

Reply:
빈 문자열 체크는 컴포넌트 반환하는 useMemo 훅 내부에서 하고 있었네요. 내부 조건문과 `activeClear` 변수가 하는 일이 겹치는 부분이 있어서, 내부 조건문을 없애고 `activeClear`에서 체크를 해도 될것 같습니다. 

```typescript
  const ClearComponent = useMemo(() => (
    <Styled.ClearIconWrapper
      onClick={handleClear}
    >
      {
        normalizedValue && normalizedValue.length > 0 && (focused || hovered) && (
          <Icon
            testId={TEXT_INPUT_CLEAR_ICON_TEST_ID}
            source={CancelCircleFilledIcon}
            size={IconSize.XS}
          />
        )
      }
    </Styled.ClearIconWrapper>
  ), [
    focused,
    handleClear,
    hovered,
    normalizedValue,
  ])

```